save-core/src/commonNonJsMain/kotlin/org/cqfn/save/core/Save.kt

```diff
@@ -95,6 +92,7 @@ class Save(
             }
         reporter.afterAll()
         reporter.out.close()
+        logInfo("Finished")
```

Comment:
Save processing finished. If the result is unexpected, please use --debug option